src/main/java/frc/robot/vision/rawdata/RawVisionAprilTagData.java

```diff
@@ -0,0 +1,24 @@
+package frc.robot.vision.rawdata;
+
+import edu.wpi.first.math.geometry.Pose3d;
+
+public class RawVisionAprilTagData extends RawVisionData {
+
+	private final double aprilTagHeight;
+	private final double distanceFromAprilTag;
+
+	public RawVisionAprilTagData(Pose3d estimatedPose, double aprilTagHeight, double distanceFromAprilTag, double timestamp) {
+		super(estimatedPose, timestamp);
+		this.aprilTagHeight = aprilTagHeight;
+		this.distanceFromAprilTag = distanceFromAprilTag;
+	}
+
+	public double aprilTagHeight() {
```

Comment:
Consider adding get to the name of the function

Reply:
I've done it that way to mimic the behavior of records. It's "immutable" object anyway

Reply:
Correct but, since it no longer is a record I have to put "get" in the methods names because its a convention